README.md

```diff
@@ -29,8 +41,8 @@ cd s2spy
 python3 -m pip install .
```

Comment:
To install the `main` branch, no need to clone the repo. It is possible to install it as: 
`python3 -m pip install git+https://github.com/AI4S2S/s2spy.git@main`

Reply:
Thanks for the suggestion! It is updated now.